models/settings/fields.yaml

```diff
@@ -0,0 +1,7 @@
+fields:
+    user_class:
+        label: "User Model Class"
+        span: auto
+        required: 1
+        type: text
+        comment: "i.e. Backend\\Models\\User. Default: LukeTowers\\Passport\\Models\\BackendUser"
```

Comment:
@b8ne if you're still interested in getting this merged in as a Winter plugin, then it would probably be better to handle this configuration in a config file rather than a DB based settings item. If not, just let me know and I'll make the necessary tweaks and transfer it over to the winter namespace.

Reply:
It actually is in a config file, see line 77 of config/config.php.  I just use the config, but thought other people may find the GUI way easier.  If we scrap the settings and just use the config, then the entire models/Settings.php implementation can also be removd.

Reply:
> Have you tested it with v1.1 (ie Laravel 6)?

Just updated dependencies and tested on Laravel 6.